src/main/java/ir/tehran/jug/projects/eventmanagement/data/entities/Organization.java

```diff
@@ -0,0 +1,38 @@
+package ir.tehran.jug.projects.eventmanagement.data.entities;
+
+import ir.tehran.jug.projects.eventmanagement.data.base.entity.BaseEntity;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.ManyToOne;
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+@Data
+@Entity
+public class Organization extends BaseEntity<Long> {
+
+    @NotBlank(message = "Name can not be empty!")
+    private String name;
+
+    @NotBlank(message = "Phone number can not be empty!")
+    @Digits(integer = 11,
+            fraction = 0,
+            message = "Phone number must contains 12 digits!")
+    @Column(name = "phone_number",
+            columnDefinition = "varchar(20)",
+            unique = true,
+            nullable = false)
+    private String phoneNumber;
```

Comment:
Do more research on using long type for phone number and its effect on database performance